packages/core/src/tracing/instrumentation/frameworks/express.js

```diff
@@ -30,18 +30,26 @@ exports.init = function init() {
 function instrument(express) {
   if (express.Router && express.Router.handle && express.Router.use) {
     // express 4
+    // capture the uncaught error
     shimmer.wrap(express.Router, 'handle', shimExpress4Handle);
     shimmer.wrap(express.Router, 'use', shimExpress4Use);
   }
 
   if (express.Route && express.Route.prototype) {
-    // express 4
+    // express v4, v5 beta
+    // capture the path template
     methods.concat('all').forEach(method => {
       if (typeof express.Route.prototype[method] === 'function') {
         shimmer.wrap(express.Route.prototype, method, shimHandlerRegistration);
       }
     });
   }
+  if (express.Router && express.Router.prototype) {
```

Comment:
In express v5 beta, the routing logic has been refactored to use the [router](https://www.npmjs.com/package/router) module instead of a locally defined one. The methods to capture uncaught errors are now on the prototype of express.Router. This ensures the code correctly wraps and intercepts the handle and use methods in Express 5 beta for error capturing.
reference https://github.com/expressjs/express/commit/cec5780db4f07a61e21e139e38af20b02dd5ae3a